### PR TITLE
add conceptual guides to new docs

### PR DESCRIPTION
**Pull request by agola11**

### CODE REVIEW

File: versioned_docs/version-2.0/concepts/admin/admin.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+API keys are being deprecated in favor of personal access tokens (PATs). We recommend using PATs for all new integrations.
+:::
+
+API keys are used to authenticate requests to the LangSmith API. They are created by users and scoped to a workspace. This means that all requests made with an API key will be associated with the workspace that the key was created in. The API key will have the ability to create, read, update, delete all resources within that workspace.
+
+API keys are prefixed with `ls__`
+
+## Personal access tokens (PATs)
+
+Personal access tokens (PATs) are used to authenticate requests to the LangSmith API. They are created by users and scoped to a user. The PAT will have the same permissions as the user that created it.
```
Comment (langchain-infra) [inline versioned_docs/version-2.0/concepts/admin/admin.mdx:29]: and is scoped to a workspace

File: versioned_docs/version-2.0/concepts/admin/admin.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Service keys are prefixed with `lsv2sk__`
+
+## Roles
+
+Roles are used to define the permissions that a user has within a workspace. There are three built-in roles:
+
+- `admin` - has full access to all resources within the workspace
+- `viewer` - has read-only access to all resources within the workspace
+- `user`
```
Comment (langchain-infra) [inline versioned_docs/version-2.0/concepts/admin/admin.mdx:45]: also i am planning on adding an annotation role: read-only permissions except ability to add feedback/update annotation queue

File: versioned_docs/version-2.0/concepts/admin/admin.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+This conceptual guide covers topics related to managing users, organizations, and workspaces within LangSmith.
+
+## Organizations
+
+An organization is a logical grouping of users within LangSmith. Typically, there is one organization per company. An organization can have multiple workspaces.
+
+## Workspaces
+
+A workspace is a logical grouping of users within an organization that share access to the same set of resources, including tracing projects, datasets, annotation queues, and prompts.
```
Comment (bvs-langchain) [inline versioned_docs/version-2.0/concepts/admin/admin.mdx:11]: nit: to make it very explicit that the resources are also tied to an individual workspace, maybe something like this:
> A workspace is a logical grouping of users and resources within an organization. Users may have permissions in a workspace that grant them access to the resources in that workspace, including tracing projects, datasets, annotation queues, and prompts.

File: versioned_docs/version-2.0/concepts/evaluation/evaluation.mdx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+This score may be based on comparing the outputs with reference outputs (e.g. with string matching or using an LLM as a judge).
+However, there are also evaluators that don't require a reference output - for example, one that checks if the output is valid JSON, which is a common requirement in LLM applications.
+LangSmith allows you to run evaluations on your application via `Datasets`, which are made up of `Examples`.
+
+The following diagram outlines the building blocks for evaluations in LangSmith. `Datasets` define the inputs over which you run your chain, model, or agent (the `Task`),
+and optionally the reference outputs against which your evaluator will compare the outputs of your `Task`. These datasets can be from any number of sources -
+you might manually curate them, collect them from user input/feedback, or generate them via LLM. Your `Evaluator` can be any arbitrary function which returns a score
+based on the inputs and outputs of your `Task`, and the reference output if desired. You can also use [one of LangSmith's off-the-shelf
```
Comment (bvs-langchain) [inline versioned_docs/version-2.0/concepts/evaluation/evaluation.mdx:15]: missing a link for `[one of LangSmith's off-the-shelf evaluators]`

File: versioned_docs/version-2.0/concepts/evaluation/evaluation.mdx [hunk 5]
```diff
@@ -0,0 +1,138 @@
+# Evaluation
+
+This conceptual guice covers topics that are important to understand when running evaluations in LangSmith.
```
Comment (bvs-langchain) [inline versioned_docs/version-2.0/concepts/evaluation/evaluation.mdx:3]:
```suggestion
This conceptual guide covers topics that are important to understand when running evaluations in LangSmith.
```

File: versioned_docs/version-2.0/concepts/tracing/tracing.mdx [hunk 6]
```diff
@@ -0,0 +1,64 @@
+import ThemedImage from "@theme/ThemedImage";
+
+# Tracing
+
+This conceptual guice covers topics that are important to understand when logging traces to LangSmith. A `Trace` is essentially a series of steps that your application takes to go from input to output. Each of these individual steps is represented by a `Run`. A `Project` is simply a collection of traces. The following diagram displays these concepts in the context of a simple RAG app, which retrieves documents from an index and generates an answer.
```
Comment (bvs-langchain) [inline versioned_docs/version-2.0/concepts/tracing/tracing.mdx:5]:
```suggestion
This conceptual guide covers topics that are important to understand when logging traces to LangSmith. A `Trace` is essentially a series of steps that your application takes to go from input to output. Each of these individual steps is represented by a `Run`. A `Project` is simply a collection of traces. The following diagram displays these concepts in the context of a simple RAG app, which retrieves documents from an index and generates an answer.
```

File: versioned_docs/version-2.0/concepts/admin/admin.mdx [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# Admin
```
Comment (bvs-langchain) [inline versioned_docs/version-2.0/concepts/admin/admin.mdx:1]: I think one operational piece we'll want to include somewhere is info on user deletion (org-level) and user removal (workspace level). Likely important to have all of the side effects laid out clearly. It's less conceptual so may belong elsewhere but wanted to flag.

File: versioned_docs/version-2.0/concepts/admin/admin.mdx [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+## Organizations
+
+An organization is a logical grouping of users within LangSmith. Typically, there is one organization per company. An organization can have multiple workspaces.
+
+## Workspaces
+
+A workspace is a logical grouping of users within an organization that share access to the same set of resources, including tracing projects, datasets, annotation queues, and prompts.
+
+## Users
```
Comment (bvs-langchain) [inline versioned_docs/version-2.0/concepts/admin/admin.mdx:13]: We may want to add a section about authentication modes, or I can add that under the self-hosted docs specifically with the basic auth launch

File: versioned_docs/version-2.0/concepts/admin/admin.mdx [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+## Service keys
+
+Service keys are similar to PATs, but are used to authenticate requests to the LangSmith API on behalf of a service account.
+
+Service keys are prefixed with `lsv2sk__`
+
+## Roles
+
+Roles are used to define the permissions that a user has within a workspace. There are three built-in roles:
```
Comment (bvs-langchain) [inline versioned_docs/version-2.0/concepts/admin/admin.mdx:41]: @langchain-infra worth adding anything here about org-level roles as well?

Comment (langchain-infra) [reply versioned_docs/version-2.0/concepts/admin/admin.mdx:41]: IMO not worth until we actually create those?

File: versioned_docs/version-2.0/concepts/admin/admin.mdx [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+## Workspaces
+
+A workspace is a logical grouping of users within an organization that share access to the same set of resources, including tracing projects, datasets, annotation queues, and prompts.
+
+## Users
+
+A user is a person who has access to LangSmith. Users can be members of one or more organizations and workspaces within those organizations.
+
+## API keys
```
[the remark this answers is not in the document]
Comment (agola11) [reply versioned_docs/version-2.0/concepts/admin/admin.mdx:17]: yes, updated.

File: versioned_docs/version-2.0/concepts/admin/admin.mdx (hunk 8 above)
Comment (agola11) [reply versioned_docs/version-2.0/concepts/admin/admin.mdx:13]: This should go under reference

File: versioned_docs/version-2.0/concepts/admin/admin.mdx (hunk 1 above)
Comment (agola11) [reply versioned_docs/version-2.0/concepts/admin/admin.mdx:29]: We talked about how this is scoped to an org now right?

**Pull request merged**